pkgs/by-name/mo/modrinth-app-unwrapped/package.nix

```
@@ -1,86 +1,55 @@
 {
   lib,
   stdenv,
-  stdenvNoCC,
   fetchFromGitHub,
   rustPlatform,
-  buildGoModule,
-  nix-update-script,
-  modrinth-app-unwrapped,
   cacert,
   cargo-tauri,
-  desktop-file-utils,
-  esbuild,
   darwin,
-  jq,
+  desktop-file-utils,
   libsoup,
-  moreutils,
-  pnpm_8,
   nodejs,
   openssl,
   pkg-config,
+  pnpm_9,
+  turbo,
   webkitgtk,
 }:
-rustPlatform.buildRustPackage {
+
+let
+  pnpm = pnpm_9;
```

```suggestion
  pnpm = pnpm_9;
  inherit (lib)
    licenses
    maintainers
    platforms
    ;
```
`with lib` is an established anti-pattern that I think we shouldn't be so fast to introduce here.